src/components/Questionnaire/ReviewFileTypeTable.tsx

```diff
@@ -77,24 +77,6 @@ const StyledTableCell = styled(TableCell)(() => ({
   },
 }));
 
```

Comment:
The last two cells had different padding, fonts, and text color. Although it's apart of the Figma mockup, I suspect it may have been an accident? Regardless, it's trivial enough that I'm removing it.

This also caused the PDF to put cells on different pages

![image](https://github.com/user-attachments/assets/ba3330f7-4b13-4ba0-b3e3-c1863ea4a63d)